<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LifeWidget</class>
 <widget class="QWidget" name="LifeWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushButton_close">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>10</y>
     <width>131</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_comfort">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>110</y>
     <width>201</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Sunscreen">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>120</y>
     <width>181</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_makeUp">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>110</y>
     <width>181</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_dresssing">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>110</y>
     <width>211</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_allergy">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>280</y>
     <width>211</width>
     <height>121</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_airquality">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>280</y>
     <width>201</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_cold">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>280</y>
     <width>181</width>
     <height>121</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_traffic">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>280</y>
     <width>181</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Travel">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>440</y>
     <width>211</width>
     <height>121</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_motion">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>450</y>
     <width>181</width>
     <height>121</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_airdrying">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>450</y>
     <width>201</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_carwash">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>450</y>
     <width>181</width>
     <height>121</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_city">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>10</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 17pt &quot;Ubuntu&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>600</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images出行助手/指数.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>pushButton_close</zorder>
  <zorder>pushButton_comfort</zorder>
  <zorder>pushButton_Sunscreen</zorder>
  <zorder>pushButton_makeUp</zorder>
  <zorder>pushButton_dresssing</zorder>
  <zorder>pushButton_allergy</zorder>
  <zorder>pushButton_airquality</zorder>
  <zorder>pushButton_cold</zorder>
  <zorder>pushButton_traffic</zorder>
  <zorder>pushButton_Travel</zorder>
  <zorder>pushButton_motion</zorder>
  <zorder>pushButton_airdrying</zorder>
  <zorder>pushButton_carwash</zorder>
  <zorder>label_city</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
